.github/workflows/ci.yaml

```diff
@@ -0,0 +1,55 @@
+name: openEQUELLA Admin Console Launcer CI
+
+on:
+  push:
+    branches:
+      - master
+      - develop
+      - release/**
+    tags:
+      - "**"
+  pull_request:
+    branches:
+      - master
+      - develop
+      - release/**
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Gradle Cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 8
+
+      - name: Build with Gradle
+        run: ./gradlew build
+
+      - name: Save artefacts
+        uses: actions/upload-artifact@v2.2.3
+        with:
+          name: Artefacts
+          path: build/distributions
```

Comment:
After the talk above about the tests, I wonder if perhaps we should save also the test report.